scripts/Diffraction/isis_powder/routines/focus.py

```diff
@@ -112,23 +98,26 @@ def _focus_one_ws(
                     Material=sample_details.generate_sample_material(),
                 )
 
-    # Align
-    mantid.ApplyDiffCal(InstrumentWorkspace=input_workspace, CalibrationFile=run_details.offset_file_path)
-    aligned_ws = mantid.ConvertUnits(InputWorkspace=input_workspace, Target="dSpacing")
-
     solid_angle = instrument.get_solid_angle_corrections(run_details.vanadium_run_numbers, run_details)
-    if solid_angle:
-        aligned_ws = mantid.Divide(LHSWorkspace=aligned_ws, RHSWorkspace=solid_angle)
-        mantid.DeleteWorkspace(solid_angle)
 
-    # Focus the spectra into banks
-    focused_ws = mantid.DiffractionFocussing(InputWorkspace=aligned_ws, GroupingFileName=run_details.grouping_file_path)
+    focused_ws = _calibrate_and_focus_workspace(
+        input_workspace, run_details.offset_file_path, run_details.grouping_file_path, solid_angle=solid_angle
+    )
 
-    instrument.apply_calibration_to_focused_data(focused_ws)
+    if instrument.get_instrument_prefix() != "OSIRIS":
+        instrument.apply_calibration_to_focused_data(focused_ws)
```

Comment:
I think this call to `apply_calibration_to_focused_data` can be moved outside of the if statement and applied to OSIRIS also

Reply:
Added